webstorage/src/main/java/WebStorageApi.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *  Copyright 2018, TeamDev. All rights reserved.
+ *
+ *  Redistribution and use in source and/or binary forms, with or without
+ *  modification, must retain the above copyright notice and the following
+ *  disclaimer.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ *  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ *  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ *  A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ *  OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ *  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ *  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ *  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ *  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ *  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ *  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import com.teamdev.jxbrowser.browser.Browser;
+import com.teamdev.jxbrowser.engine.Engine;
+import com.teamdev.jxbrowser.engine.EngineOptions;
+import com.teamdev.jxbrowser.frame.WebStorage;
+import com.teamdev.jxbrowser.navigation.event.FrameLoadFinished;
+
+import static com.teamdev.jxbrowser.engine.RenderingMode.OFF_SCREEN;
+
+/**
+ * This example demonstrates how to access the local/session storage and perform operations with it.
+ */
+public class WebStorageApi {
+
+    public static void main(String[] args) {
+        Engine engine = Engine.newInstance(
+                EngineOptions.newBuilder(OFF_SCREEN).build());
+        Browser browser = engine.newBrowser();
+
+        browser.navigation().on(FrameLoadFinished.class, event -> {
+            WebStorage webStorage = event.frame().localStorage(); // localStorage access
+//            WebStorage webStorage = event.frame().sessionStorage(); // sessionStorage access
+            String key = "your-custom-key";
+            webStorage.putItem(key, "Hello!");
+            System.out.println(webStorage.item(key).orElseThrow(IllegalStateException::new));
+        });
+
+        browser.navigation().loadUrl("https://www.teamdev.com");
```

Comment:
I think you do not need to load a web page to show how to work with web storage.